webApp/src/main/java/com/infoshare/formobjects/VolunteerForm.java

```diff
@@ -61,8 +78,14 @@ public void setTypeOfHelp(TypeOfHelp typeOfHelp) {
         this.typeOfHelp = typeOfHelp;
     }
 
-    public boolean isAvalible() {
-        return isAvalible;
+    public boolean getIsAvailable() { return isAvailable; }
```

Comment:
przy metodach zwracających boolean stosuje się w nazwie is, should, has. Było dobrze wcześniej :)
```suggestion
    public boolean isAvalible() {
        return isAvalible;
```

Reply:
dziękuję, poprawione :+1: